docs/content/docs/getting-started/build.md

```diff
@@ -154,11 +154,11 @@ cd LocalAI
 # build the binary
 make build
 
```

Comment:
updated to `phi-2.Q2_K` as requested in the issue, it has a smaller size, easier to download